src/introspection/buildQueriesForResource/buildFieldList.js

```diff
@@ -1,20 +1,79 @@
+import { TypeKind } from 'graphql';
 import isNotGraphqlPrivateType from '../isNotGraphqlPrivateType';
 
-export default (resource, type, { excludeFields }) =>
-    resource.fields
+export const isSubObject = field => {
+    if (
+        field.type.kind === TypeKind.OBJECT ||
```

Comment:
either split the condition in several if for better readbility, or return it directly.